packages/integrations/svelte/client.js

```diff
@@ -104,9 +104,15 @@ function finishUsingConsoleFilter() {
  */
 function filteredConsoleWarning(msg, ...rest) {
 	if (consoleFilterRefs > 0 && typeof msg === 'string') {
-		// Astro passes a `class` prop to the Svelte component, which
+		// Astro passes `class` and `data-astro-cid` props to the Svelte component, which
 		// outputs the following warning, which we can safely filter out.
-		const isKnownSvelteError = msg.endsWith("was created with unknown prop 'class'");
+
+		// NOTE: In practice data-astro-cid props have a hash suffix. Hence the use of a
+		// quoted prop name string without a closing quote.
+
+		const isKnownSvelteError =
+			msg.endsWith("was created with unknown prop 'class'") ||
+			msg.indexOf("was created with unknown prop 'data-astro-cid") > 0;
```

Comment:
Can we use a `.includes` instead of `.indexOf > 0`? It's a bit more succinct.

Reply:
Updated to use includes.